### ISSUE TITLE
Can you switch to the new format while you have CDN requests pending from the old?

### ISSUE THREAD

**Issue by Satyam** [yui/yogi#10]
I have a few CDN requests on a couple of my modules.  If I do a yogi update, will they be affected?  So far I created the new repository, made my local copy and now copied my modules from the old folder to the new.  I stopped at doing the yogi update on each.   If I change the repo and path via yogi, will it all still work?

Also, once I do this if I make changes on the new version, will there be any conflicts with the current CDN requests that will be pulled from the old repository?

In other words, can I go ahead or I should I wait until Tuesday?

Thanks

**Comment by davglass** [comment]
There will be no CDN deployments until at least Nov 28th due to the
holiday (and I'm on vacation as of today).

**Comment by Satyam** [comment]
Enjoy your vacation, you deserve it and I hope you don't even read this until back.

When you do, I will probably have further changes on some of the modules.  Actually, I ventured doing the `yogi update` on the new repository, I assumed it won't affect anything on-line.  I didn't change the repo or path with `yogi info` since I assumed that would make changes beyond my machine.

So, how far can I go when I still have CDN requests pending from the old repository, the fork from the full yui3-gallery?  Shall I wait until those are processed before doing anything with the new?  Can I change the repo and path and do further CDN requests?

In the meantime I'll keep working locally on my machine.  

Enjoy your vacation!!!

**Comment by davglass** [comment]
If you change the repo and path, then the next build will attempt to use them from that new path (even if you already submitted a CDN request since the info is related to the module and not the request).

I would recommend not changing the path or repo until you are ready to submit the cdn request.
